app/components/Dropdown/Dropdown.js

```diff
@@ -0,0 +1,205 @@
+
+import React, { Component } from 'react';
+import Baseline from 'react-baseline';
+import classnames from 'classnames';
+
+import GridContainer from 'GridContainer/GridContainer';
+import SandboxPreview from 'SandboxPreview/SandboxPreview';
+import SandboxTogglePanel from 'SandboxTogglePanel/SandboxTogglePanel';
+import SandboxToggle from 'SandboxToggle/SandboxToggle';
+import Section from 'Section/Section';
+import HeadlineText from 'HeadlineText/HeadlineText';
+import Spec from 'Spec/Spec';
+import Code from 'Code/Code';
+
+import SeekDropDown from 'seek-style-guide/react/fields/Dropdown/Dropdown';
+
+const specs = {
+  default: {
+    Height: '5 grid rows',
+    'Internal gutters': '15px — @field-gutter-width',
+    'Margin bottom': '3 grid rows',
+    'Text color': '@sk-charcoal',
+    'Font scale': '1.8 — @interaction-type-scale',
+    Border: '1px — @sk-mid-gray-light',
+    'Border radius': '2px — @field-border-radius'
+  },
+  focus: {
+    Border: '1px @sk-focus'
+  },
+  invalid: {
+    Border: '1px @sk-pink'
+  }
+};
+
+function getSpec(specsObj) {
+  return Object.keys(specsObj).reduce((result, specName) => ({
+    ...result,
+    ...(specsObj[specName] ? specs[specName] : {})
+  }), {});
+}
+
+export default class DropDownDemo extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      focus: false,
+      invalid: false,
+      baseline: false,
+      help: false,
+      inputValue: ''
+    };
+
+    this.toggleFocus = this.toggleFocus.bind(this);
+    this.toggleInvalid = this.toggleInvalid.bind(this);
+    this.toggleBaseline = this.toggleBaseline.bind(this);
+    this.toggleHelp = this.toggleHelp.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  toggleFocus(event) {
+    this.setState({
+      focus: event.target.checked
+    });
+  }
+
+  toggleInvalid(event) {
+    this.setState({
+      invalid: event.target.checked
+    });
+  }
+
+  toggleHelp(event) {
+    this.setState({
+      help: event.target.checked
+    });
+  }
+
+  toggleBaseline(event) {
+    this.setState({
+      baseline: event.target.checked
+    });
+  }
+
+  handleChange(event) {
+    this.setState({
+      inputValue: event.target.value
+    });
+  }
+
+  render() {
+    const { focus, invalid, baseline, help, inputValue } = this.state;
+    const className = classnames({
+      [styles.input]: true,
+      [dropdownStyles.rootFocus]: focus
+    });
+    const spec = getSpec({
+      default: true,
+      focus,
+      invalid
+    });
+    const options = [
+      {
+        value: '1',
+        label: 'Developer'
+      },
+      {
+        value: '2',
+        label: 'Tester'
+      },
+      {
+        value: '3',
+        label: 'Product Manager'
+      }
+    ];
+    const dropdown = (
+      <SeekDropDown
+        id="jobTitles"
+        className={className}
+        invalid={invalid}
+        label="Job Titles"
+        focus={focus}
```

Comment:
`focus` does not appear necessary to pass as a prop (also it's not validated or consumed internally to `Dropdown`)